oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+Whether or not a GitHub Issue is used, a GitHub Pull Request will exist for creation/merging of the ADR.  This is the primary medium for the Advisory Process, and where we expect conversations will take place.  By the time a pull request is created, we expect the decision maker will have written down the context of the decision - necessary to orient advisors to the problem space - as well as draft of their potential decision.
+
+Decision Maker's Checklist
+--------------------------
+
+In order to help ADR authors ensure they get advice from the proper set of stakeholders, we will create a checklist of possible groups/individuals to choose from.  It's the responsibility of the author to ensure the relevant people are aware of the ADR and are given the opportunity to provide feedback.
+
+The Decision Maker's Checklist is a document that a decision authors can review to ensure they're seeking advice from the proper stakeholders and experts when authoring an ADR. It's that individual's responsibility to select all those stakeholders and experts (which may be individuals or groups) that are impacted by their decision based on their understanding of the scope.  Other individuals or groups may be added later as the scope and impact is better understood.
+
+GitHub Groups
+^^^^^^^^^^^^^
+
+As a companion to the Decision Maker's Checklist, a set of GitHub groups will be created in the ``openedx`` organization that can be tagged on Pull Requests for ADRs.  The Decision Maker's Checklist will indicate the GitHub handles of individuals, or of these groups for use in creating pull requests.
+
+Checklist Organization
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Decision Maker's Checklist is a reference, and so decision maker's will be able to look up individuals by various criteria.  To this end, the checklist will list stakeholders and experts for products, domains, and by role, at a minimum.  This means that individuals or groups may appear more than once if they're experts on several different domains, or have multiple roles, for instance.
+
+We will necessarily iterate on the organization of the checklist as we learn more about how we use it.
+
+Technical Strategic Review (WIP)
+--------------------------------
+
+Coinciding with an organization's broader product strategy and prioritization cadence, establish quarterly technical strategic reviews focused on engineering and architectural priorities as they relate to our businesses' and platform's needs.  Depending on the size and team topologies of a given organization, The number of such meetings may vary.
+
+There may several reviews, and they may be domain/theme-specific depending on the organization.  This process is not prescriptive on the number or cadence of review ceremonies, except to say that they should happen as frequently as an organization's business/product planning process occurs.
+
+The review should be attended by engineering managers and at least one technical lead from each team in the group.
```

Comment:
I'm not sure I understand the intention of this section. I know it's a WIP so I'll wait for it to be fleshed out but my primary source of confusion is it seems as if both one org runs it on their cadence, but all orgs must attend?

Reply:
I'm deleting this section. :)  See above comment; it's not baked enough to codify.